dot/network/message.go

```diff
@@ -48,6 +48,8 @@ const (
 	RemoteHeaderResponseType  = 11
 	RemoteChangesRequestType  = 12
 	RemoteChangesResponseType = 13
+	RemoteReadChildRequestType = 14
+	RemoteReadChildResponseType = 15
```

Comment:
is this from anywhere? in general, message IDs are deprecated, so it would be better not to have them

Reply:
looking at the rust code it seems like the message IDs for light messages are as follows: https://github.com/paritytech/substrate/blob/cf2f6699ab387db96e6c0301d2178190b5559a3e/client/network/src/schema/light.v1.proto#L16

could you update the message IDs?